### ISSUE TITLE
Unable to set custom currency symbol 2.1.3

### ISSUE THREAD

**Issue by maxxis2112** [magento/magento2#7878]
After upgrading to 2.1.3 I am unable to set a custom currency symbol.

Deselecting "Use Standard" and entering "R" instead of "ZAR" saves and shows "ZAR again in the field with "Use Standard" selected.

Change is not reflected on the site after clearing cache. 

### Preconditions
Production mode
Magento 2.1.3

### Steps to reproduce
1.  Stores -> Currency Symbols
2. Deselect "Use Standard"
3. Enter new symbol
4. Save
5. Flush Cache

### Expected result

1. Currency symbol should update to "R".

### Actual result

1. Symbol stays the same at "ZAR"

**Comment by maxxis2112** [comment]
Still having no luck resolving this issue. Unfortunately no options to roll back either to 2.1.2.

If anybody has any interim solutions i would really appreciate it.

**Comment by veloraven** [comment]
@maxxis2112 thank you for you report.
I was able to reproduce this issue only partially.
Could you please specify:
* in the 'Actual result' - where exactly the currency symbol is not changed? (you can add a screenshot)
* was it single-store mode?

**Comment by maxxis2112** [comment]
We are using single store mode yes. When setting to not be in single store
mode it accepts it.

On the front end the currency is shown as ZAR
We want it to show as R
So we change the custom currency symbol to R in admin side but it isnt
accepting it.

Johan

**Comment by maxxis2112** [comment]
Once you disable single store mode you can set the correct custom currency symbol and then switch the store back to single store mode it will keep the currency symbol

**Comment by Koc** [comment]
Have same issue on 2.1.3.

**Comment by MrDaar** [comment]
@maxxis2112 @veloraven 

I'm not sure if this is the same issue, but I've been unable to save custom currencies.

It looked as though the custom symbol was being used as some sort of key, and if the symbol matched the key, the corresponding symbol would be used, and the value set back to "default" rather than using the actual value that was set.

My fix was to force the value in config, either in database or in config.xml:
```
<?xml version="1.0"?>
<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
    <default>
        <currency>
            <options>
                <customsymbol>a:4:{s:3:"DKK";s:4:" DKK";s:3:"EUR";s:4:" EUR";s:3:"NOK";s:4:" NOK";s:3:"SEK";s:4:" SEK";}</customsymbol>
            </options>
        </currency>
    </default>
</config>
```

Notice that the keys and values are the same in the serialised array, so forcing the currency to take the new symbol.

Previously specifying SEK/NOK/DKK would always return "kr" as the currency symbol, and reset to "default".

**Comment by bjornet** [comment]
Have same issue on 2.1.2.

**Comment by jeffreytu** [comment]
Hello. I'm having this issue on 2.0.7. I originally had it on "Use Standard" which was "$" for my store, then put "USD" and saved. Now I want to switch back to "Use Standard" and saved but it reverts back to "USD". Cache was cleared. 

Anyone know the table where this custom currency is so I can manually change it for now? Thank you.

**Comment by zeeshan2523806** [comment]
Thanks @MrDaar, I am having exactly the same issue with same currencies, unable to edit or uncheck the standard box from the currency symbols. currency is being showing "kr" on all the store, either we choose SEK, DKK or NOK. any idea where we have to update this config.xml file ? or if have to update in the DB than which tables and field ?

**Comment by magento-engcom-team** [comment]
@maxxis2112, thank you for your report.
The issue is already fixed in 2.1.8
